src/mysky/utils.test.ts

```diff
@@ -125,41 +125,20 @@ const portal2Urls = combineStrings(["", "http://", "https://", "HTTPS://"], ["si
 
 // TODO: Test cases with portal servers.
 describe("getRedirectUrlOnPreferredPortal", () => {
-  let mock: MockAdapter;
-
-  beforeEach(() => {
-    mock = new MockAdapter(axios);
-  });
-
-  const portal1SkappUrls = combineStrings(["", "https://"], ["skapp.hns.siasky.net"], ["", "/"]);
-  const portal2SkappUrls = combineStrings(["", "https://"], ["skapp.hns.siasky.xyz"], ["", "/"]);
-
-  const cases: Array<[string, string, string, string]> = [
+  const cases: Array<[string, string, string]> = [
     // Test redirecting from one portal to another.
-    ...(composeTestCases(combineArrays(portal1SkappUrls, portal2Urls), "https://skapp.hns.siasky.xyz").map(
-      ([[a, b], c]) => ["siasky.net", a, b, c]
-    ) as [string, string, string, string][]),
-    ...(composeTestCases(combineArrays(portal2SkappUrls, portal1Urls), "https://skapp.hns.siasky.net").map(
-      ([[a, b], c]) => ["siasky.xyz", a, b, c]
-    ) as [string, string, string, string][]),
+    ...(composeTestCases(combineArrays(["skapp.hns"], portal2Urls), "https://skapp.hns.siasky.xyz").map(
```

Comment:
this is neat but just writing all the cases in a table is much easier on the eyes IMO

Reply:
+1 for table tests

Reply:
For sure, that is more prone to human error though. When I get a chance, it would be good to have a comment which enumerates all the cases.